package.json

```diff
@@ -60,6 +60,9 @@
   "workspaces": {
     "packages": [
       "packages/*"
+    ],
+    "nohoist": [
+      "**/html-minifier-terser"
```

Comment:
Can you leave a link to a relevant docusaurus GH issue on this PR?

Reply:
https://github.com/facebook/docusaurus/issues/3515#issuecomment-707145367